demo/package.json

```diff
@@ -5,6 +5,6 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/vaadin/gwt-polymer-elements.git",
   "dependencies": {
-    "gwt-api-generator": "vaadin/gwt-api-generator#master"
+    "gwt-api-generator": "vaadin/gwt-api-generator#elemental2"
```

Comment:
This should be changed before merging. Leave it as it is for now.

Reply:
ok then i will change it back to elemental2
and the pom file should i change how it is generated by the gwt-api-generator?

Reply:
this one is done